app/components/quiz/Quiz.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import AppStore from 'store/appStore';
+import QuizQuestion from 'components/quiz/QuizQuestion.js';
+
+import '../../styles/quiz.scss';
+
+export default React.createClass({
+  displayName: 'Quiz',
+
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
+  propTypes: {
+    questions: React.PropTypes.array.isRequired
+  },
+
+  getDefaultProps: function() {
+    return {
+      questions: [
+        {
+          id: 1,
+          text: 'Dummy question',
+          context: {
+            text: 'Dummy context for display purposes only',
+            highlights: [[6, 34]]
+          },
+          answers: [
+            { text: 'a', question: 1, type: 'radio'},
+            { text: 'b', question: 1, type: 'radio'},
+            { text: 'c', question: 1, type: 'radio'},
+            { text: 'd', question: 1, type: 'radio'}
+          ]
+        }
+      ]
+    };
+  },
+
+  getInitialState: function() {
+    return {value: null};
+  },
+
+  onUpdate: function(value) {
+    this.setState({value});
```

Comment:
also i'm wondering if i shouldn't move this to redux.js as i've heard good things.